vision/ui/main_window.py

```diff
@@ -0,0 +1,67 @@
+from tkinter import*
+ 
+# creating button for autonomous mapping
+window = Tk()
+window.title("welcome")
+window.geometry('300x300')
+window.configure(width =50,height = 50, bg='blue')
+ 
+lbl = Label(window, text = "Autonomous Mapping",  bg = 'red', fg ='white')
+lbl.grid(column=0, row=0)
+def clicked():
+    lbl.configure(text="Autonomous Mapping (Clicked)")
```

Comment:
I think it would be best to keep the label text the same. For now you can replace these methods with `pass` which will tell python to do nothing when it runs the method. Once the other code gets merged in, you can call it from here.